Communications/Server.cs

```diff
@@ -121,6 +150,25 @@ private static void WaitForClientsTCP(object ReceivePort)
             TCPListener.Stop();
         }
 
+        /// <summary>
+        /// Create a new sending buffer for client if the client is new.
+        /// </summary>
+        /// 
+        /// <param name="ClientName"> Name of the client. </param>
+        private static void CreatBufferIfClientIsNew(string ClientName)
+        {
+            lock (SendQueues)
+            {
+                if (!SendQueues.ContainsKey(ClientName))
+                {
+                    if (PriorityQueueEnabled)
```

Comment:
Single-line brackets

Reply:
Sometimes it will make a line of code too long if a single-line statement is merged with the if-statement (exceed 100 characters). It will make codes look crowded as well. So I think maybe it will make codes more readable to put the statements on separate lines.

Reply:
We don't have a line length max (and we've got tons of lines longer than 100 or even 150 characters). With modern widescreen displays, it doesn't really make sense to me to limit line length anymore.
I will usually only split the line if the if statement has complex logic (like here for example: https://github.com/huskyroboticsteam/Scarlet/blob/master/IO/BeagleBone/BBBPinManager.cs#L260 ). Other than that ,feel free to make it as long as you want :)